src/components/button-toggle/button-toggle.style.ts

```diff
@@ -136,7 +136,63 @@ const StyledButtonToggle = styled.button<StyledButtonToggleProps>`
         }
       }
       cursor: not-allowed;
-    `};
+    `}
+
+      & ${StyledIcon} {
+    color: var(--colorsActionMinor500);
+    :not([aria-pressed="true"]):not(:disabled):hover {
+      color: var(--colorsActionMinorYang100);
+    }
+  }
+
+  color: var(--colorsActionMinor500);
+  border: 1px solid var(--colorsActionMinor500);
+
+  :not([aria-pressed="true"]):not(:disabled):hover {
+    color: var(--colorsActionMinorYang100);
+    background-color: var(--colorsActionMinor600);
+    ${StyledIcon} {
+      color: var(--colorsActionMinorYang100);
+    }
+  }
+
+  &:not(:disabled):hover {
+    background-color: var(--colorsActionMinor600);
+    color: var(--colorsActionMinorYang100);
+
+    ${StyledIcon} {
+      color: var(--colorsActionMinorYang100);
+    }
+  }
+
+  &[aria-pressed="true"]:not(:hover) {
+    box-shadow: inset 0px 0px 0px 3px var(--colorsActionMinor500);
+    background-color: transparent;
+    :focus {
+      box-shadow: inset 0px 0px 0px 3px var(--colorsActionMinor500),
+        0px 0px 0px var(--borderWidth300) var(--colorsSemanticFocus500),
+        0px 0px 0px var(--borderWidth600) var(--colorsUtilityYin090);
+    }
+  }
+
+  border: 1px solid var(--colorsActionMinor500);
```

Comment:
comment: another place where we have the duplicate border definition and should be removed